src/hooks/result/useSelectBoxFilter.ts

```diff
@@ -14,7 +14,7 @@ export type ApplyPriceType = {
 }
 
 type ReturnType = {
-  tradePeriodItems: any
+  tradePeriodItems: typeof TRADE_TYPES
```

Comment:
```suggestion
tradePeriodItems: TradeTypes
```
여기도 타입 쓰면 좋을 것 같습니다!